services/mock-service/src/fsp-integration/intersolve-visa/intersolve-visa.mock.service.ts

```diff
@@ -550,4 +550,62 @@ export class IntersolveVisaMockService {
       status: HttpStatus.OK,
     };
   }
+
+  public linkToken(
+    _childTokenCode: string,
```

Comment:
We should not leave _ in finished code?

Reply:
Some of the parameters of the controllers are unused since this is a mock controller. I think leaving them there makes it a bit more explicit that they are definend but unused